### PR TITLE
Read-only controllers

### PR DESCRIPTION
**Pull request by nim-odoo**
First pass on the most used controllers which can be flagged as read-only.

### CODE REVIEW

File: addons/portal/controllers/portal.py [hunk 1]
```diff
@@ -167,7 +167,7 @@ def _prepare_home_portal_values(self, counters):
         """
         return {}
 
-    @route(['/my/counters'], type='json', auth="user", website=True)
```
Comment (nim-odoo) [inline addons/portal/controllers/portal.py:170]: ok

File: addons/mail/controllers/thread.py [hunk 2]
```diff
@@ -11,7 +11,7 @@
 
 
 class ThreadController(http.Controller):
-    @http.route("/mail/thread/data", methods=["POST"], type="json", auth="user")
```
Comment (nim-odoo) [inline addons/mail/controllers/thread.py:14]: ok

File: addons/hr_attendance/controllers/main.py [hunk 3]
```diff
@@ -174,7 +174,7 @@ def systray_attendance(self, latitude=False, longitude=False):
         employee._attendance_action_change(geo_ip_response)
         return self._get_employee_info_response(employee)
 
-    @http.route('/hr_attendance/attendance_user_data', type="json", auth="user")
```
Comment (nim-odoo) [inline addons/hr_attendance/controllers/main.py:177]: ok

**Comment by nim-odoo** [comment]
robodoo rebase-ff

**Review by seb-odoo — APPROVED** [review]

**Comment by Julien00859** [comment]
In stable?!

**Review by odony — APPROVED** [review]
nit: do you plan to squash before merging?

**Comment by nim-odoo** [comment]
> In stable?!

To complete @odony 's answer, I would also argue we are probable the only ones ~~using~~ testing the feature. As far as I understand it, there should not be any issue since there will be a rollback and replayed on the RW cursor.

Moreover, I think it is typically something we want to do in stable since we will gather the data once the Odoo version is in production on our side.

**Review by Julien00859 — APPROVED** [review]

**Comment by nim-odoo** [comment]
robodoo r+

**Pull request closed without merging**